scripts/mkcloudhost/cloudfunc

```diff
@@ -10,6 +10,12 @@ function cloudadminnet()
     echo -n 192.168.$((255-2*n))
 }
 
+function cloudironicnet()
+{
+    n=$1
+    echo -n 192.168.$((4*n))
```

Comment:
Lets wait for @bmwiedemann and/or @jdsn. They might have some better (non-conflicting) ideas.